presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -65,11 +65,32 @@ Date and Time Functions
 
 .. function:: from_iso8601_timestamp(string) -> timestamp with time zone
 
-    Parses the ISO 8601 formatted ``string`` into a ``timestamp with time zone``.
+    Parses the ISO 8601 formatted date and time ``string``, optionally with
+    time and timezone, into a ``timestamp with time zone``. Missing
+    information is defaulted to zero and local information::
+
+        SELECT from_iso8601_timestamp('2020-05-11');
```

Comment:
We should format like the other examples:
```sql
SELECT from_iso8601_timestamp('2020-05-11'); -- 2020-05-11 00:00:00.000 America/Vancouver
SELECT from_iso8601_timestamp('2020-05-11T11:15:05'); -- 2020-05-11 11:15:05.000 America/Vancouver
SELECT from_iso8601_timestamp('2020-05-11T11:15:05.055+01:00'); -- 2020-05-11 11:15:05.055 +01:00
```
The advantage of this formatting is that it's consistent and fully executable SQL (you can copy/paste into the CLI or another tool).

Reply:
I used the same setup as in the Time Zone Conversion section above on the same page ...imho it should at least be in a separate line.. I did that for all now